website/docs/reference/resource-configs/bigquery-configs.md

```diff
@@ -895,16 +895,23 @@ Find more information about materialized view limitations in Google's BigQuery [
 
 ## Python model configuration
 
-The `dbt-bigquery` adapter uses BigQuery Dataframe or Dataproc to run Python models. This process reads data from BigQuery, computes it either natively with BigQuery Dataframe or Dataproc, and writes the results back to BigQuery.
-
 **Submission methods:**
-BigQuery supports a few different mechanisms to submit python code, each with relative advantages. 
+BigQuery supports a few different mechanisms to submit Python code, each with relative advantages. The `dbt-bigquery` adapter uses BigQuery DataFrames (BigFrames) or Dataproc. This process reads data from BigQuery, computes it either natively with BigQuery DataFrames or Dataproc, and writes the results back to BigQuery.
 
-- BigQuery Dataframes (BigFrames) : Can execute pandas and scikit. There's no need to manage infrastructure and leverages BigQuery distributed query engines. It's great for analysts, data scientists, and ML engineers who want to manipulate big data using a pandas-like syntax.
+<Tabs
+  defaultValue="dataframes"
+  values={[
+    { label: 'BigQuery DataFrames', value: 'dataframes', },
+    { label: 'Dataproc', value: 'dataproc', },
+  ]
+}>
+<TabItem value="dataframes">
+
+BigQuery DataFrames can execute pandas and scikit. There's no need to manage infrastructure and leverages BigQuery-distributed query engines. It's great for analysts, data scientists, and ML engineers who want to manipulate big data using a pandas-like syntax.
```

Comment:
* It looks like the module name is [scikit-learn](https://github.com/scikit-learn/scikit-learn)
* Does ML == machine learning? or something else?
```suggestion
BigQuery DataFrames can execute pandas and scikit-learn. There's no need to manage infrastructure and leverages BigQuery-distributed query engines. It's great for analysts, data scientists, and machine learning engineers who want to manipulate big data using a pandas-like syntax.
```